_data/projects/Hacktoberfest1.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
I noticed https://github.com/ows-ali/Hacktoberfest/issues/575 was opened by @hacktoberfest-team - please remove this file from the PR